ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/name-servers.j2

```diff
@@ -1,6 +1,6 @@
 {# eos - name-server #}
-{% if name_server is arista.avd.defined %}
-{%     for node in name_server.nodes | sort %}
+{% for node in name_server.nodes | arista.avd.natural_sort %}
+{%     if name_server.source.vrf is arista.avd.defined %}
 ip name-server vrf {{ name_server.source.vrf }} {{ node }}
-{%     endfor %}
-{% endif %}
\ No newline at end of file
+{%     endif %}}
```

Comment:
one extra `}` at the end of this line. You can see in the molecule files that they suddenly produced an `}` :-)